### PR TITLE
[PR-20712] Update Shopify subtotal tracking logic

### PR DESCRIPTION
**Pull request by Doimhneacht**
## Demo

https://void-docs.talkable.com/integration/ecommerce_platforms/shopify.html#manual-integration
https://void-docs.talkable.com/advanced_features/shopify_purchase_syncing.html#shopify-webhook-payload-and-talkable-purchase-attributes-mapping

## Related Stories

[](https://talkable.atlassian.net/browse/PR-20712)

### CODE REVIEW

File: source/advanced_features/shopify_purchase_syncing.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ Talkable uses next mapping when syncing purchases from Shopify Webhook payload:
   =========================== =====================================================
   **Order Number**            `payload.id` (Shopify Purchase ID)
   **Email**                   `payload.email`
-  **Subtotal**                `payload.total_price`
+  **Subtotal**                `payload.subtotal_price`
```
[the remark this answers is not in the document]
Comment (Doimhneacht) [reply source/advanced_features/shopify_purchase_syncing.rst:40]: Kind of, but we have a simplified map here. For example, it was never `total_price`, it was `total_price_set.presentment_money.amount`, so I just followed with that logic.

[the remark this answers is not in the document]
Comment (dgtv) [reply source/advanced_features/shopify_purchase_syncing.rst:40]: I think it was just a mistake when we introduced currencies: we forgot to update this file

File: source/advanced_features/shopify_purchase_syncing.rst [hunk 2]
```diff
@@ -37,10 +37,11 @@ Talkable uses next mapping when syncing purchases from Shopify Webhook payload:
   =========================== =====================================================
   **Order Number**            `payload.id` (Shopify Purchase ID)
   **Email**                   `payload.email`
-  **Subtotal**                `payload.total_price`
-  **Taxes**                   `payload.total_tax`
-  **Discount Amount**         `payload.total_discounts`
-  **Shipping Cost**           `payload.total_shipping_price_set`
+  **Currency**                `payload.presentment_currency`
```
Comment (dgtv) [inline source/advanced_features/shopify_purchase_syncing.rst:40]: Looks like we should also update this from `currency_code` to `payload.subtotal_price_set.presentment_money.currency_code` WDYT?

Comment (Doimhneacht) [reply source/advanced_features/shopify_purchase_syncing.rst:40]: It's the same thing, so what is the profit?

> `presentment_currency`
The presentment currency that was used to display prices to the customer.

Comment (dgtv) [reply source/advanced_features/shopify_purchase_syncing.rst:40]: ok, looks like no profit them. For me it just looks more bulletproof to get it from the same object `presentment_money` which we use for `amount` so they will definitely be in sync.

**Review by dgtv — APPROVED** [review]

**Pull request merged**